### PR TITLE
add loan show page css and clean up duplicate css

### PR DESCRIPTION
**Pull request by mbburch**

### CODE REVIEW

File: db/seeds.rb [hunk 1]
```diff
@@ -4,7 +4,7 @@
                     avatar_file_name: "http://i.dailymail.co.uk/i/pix/2011/08/01/article-2021108-0D40723300000578-13_634x383.jpg")
 
 loan2 = Loan.create(title: "Kitten Mittens",
-                    description: "Keep your kittens cozy.",
+                    description: "Keep your kittens cozy. These mittens are handcrafted by a Capitol Hill artisan who anticipates being able to repay the loan within 6 months of starting her business. She already has a proven customer base.",
```
Comment (mbburch) [inline db/seeds.rb]: I left this too long on purpose. When I tried to shorten it, the spacing got weird. Feel like with seed data, sometimes it just needs to be this way.

**Comment by mbburch** [comment]
Hi guys. OK, so I added a longer description to kitten mittens in the seeds to make sure I could get one to display. I added scroll bars to the thumbnails and deleted some extraneous and duplicate CSS. Things still need neatening. I’d say some front end and functional priorities: Fix prices, make carousel photos link to loans (harder than I thought it would be. Not sure how to do it), fix sizing of the damn images!

File: db/seeds.rb (hunk 1 above)
Comment (jbrr) [inline db/seeds.rb]: That's fine, I don't really see a good way around this. Seed data is just supposed to mimic a database, so I don't really see the need for it to follow Ruby style exactly.

**Comment by jbrr** [comment]
Alright, I think this all looks good. Great job! Woof!

**Pull request merged**